### ISSUE TITLE
Double greeting message with DirectLine

### ISSUE THREAD

**Issue by lshade** [microsoft/BotFramework-Composer#8625]
## Describe the bug

I have deployed a Composer bot to Azure and configured the Direct Line channel. In Composer, I use the standard greeting message in the Conversation Update trigger.

When testing with Direct Line, the greeting message activity comes through twice. Once before the conversationUpdate, and once after. Should this only be sent after the conversationUpdate?

## Version

v2.1.0

## Browser

I'm using Postman to test this.

## OS

- [ ] macOS
- [X] Windows
- [ ] Ubuntu

## To Reproduce

In Postman, I send the following requests in this order:
1. Start a conversation (with POST to `https://directline.botframework.com/v3/directline/conversations`)
2. Send conversation update activity (with POST `https://directline.botframework.com/v3/directline/conversations/{{conversationId}}/activities`)
3. Receive activities (with GET `https://directline.botframework.com/v3/directline/conversations/{{conversationId}}/activities`) 

The last request returns 3 activities:
1. Welcome message activity
2. Conversation update activity
3. Welcome message activity again

## Expected behavior

Welcome message activity should only come after the conversationUpdate.

**Comment by dmvtech** [comment]
> I have deployed a Composer bot to Azure and configured the Direct Line channel. In Composer, I use the standard greeting message in the Conversation Update trigger.
> 
> When testing with Direct Line, the greeting message activity comes through twice.

Just to confirm; the out-of-box Greeting checks to see whether the message is to/from the bot. Can you confirm your welcome is the same as that:

![image](https://user-images.githubusercontent.com/43397658/131715213-ef4ee22d-c032-49ae-94bc-ee6c4549d949.png)

**Comment by lshade** [comment]
@dmvtech yes it's the same, I only changed the text in the `Send a response`

**Comment by dmvtech** [comment]
@lshade This only happens in a Directline client (that you created)? It doesn't reproduce with Webchat, Emulator or other channels, correct?

**Comment by lshade** [comment]
@dmvtech correct, only with Directline. We're seeing it in postman as well as our Directline client app

**Comment by lshade** [comment]
The issue ended up being that we were specifically sending `conversationUpdate` activities from the client to get back the welcome, separate from how the channel does its own `conversationUpdate`. Switching to a different activity type for the first activity will fix this
